### ISSUE TITLE
Add stochastic forcing

### ISSUE THREAD

**Issue by crocha700** [pyqg/pyqg#85]
This would be useful for problems in which we want to force the system without imposing a background velocity.

**Comment by francispoulin** [comment]
Agreed! I have been in contact with Geoff and I was hoping to try and reproduce some of his plots from his textbook on forced one-layer QG turbulence.  It shouldn't be too difficult and would be a lot of fun.  Unfortunately, I'm teaching 2 courses this term but will try and do what i can.

**Comment by rabernat** [comment]
:+1: But maybe this is beyond 0.2?

**Comment by crocha700** [comment]
Wallis & Maltrud was exactly the paper I had in mind @francispoulis.

Fine with me to add latter. Do we have a due date for 0.2?
